kubernetes/flint-api/README.md

```diff
@@ -0,0 +1,53 @@
+# REST API FLINT.Example integration for Kubernetes
+
+This Kubernetes-based setup provisions a local installation of FLINT.Example REST API on Minikube which uses a local `flint-api` image on your machine.
+
+## Installation
+
+1. [Install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) to run Kubernetes locally.
+
+2. Start your cluster on minikube, by running the command:
+	 ```sh
+	 minikube start
+	 ```
+	 It starts a local Kubernetes cluster.
+3. Before starting the local dev/test deployment run:
+	```sh
+	eval $(minikube docker-env)
```

Comment:
In the case of Windows this would be: & minikube -p minikube docker-env --shell powershell | Invoke-Expression